first/frc/index.md

```diff
@@ -2,7 +2,7 @@
 title: FRC
 ---
 ## *FIRST* Robotics Competition
-<img class="float-left" alt="FRC Logo" style="width:275px;" src="{{ site.url }}/assets/img/FIRST-logos/FRC-logo.png"/> *FIRST* Robotics Competition is for kids from grades 9-12. While each team has students as its members, they also have coaches and mentors who help as needed. A motto created by our team that encapsulates this idea is "Mentor nurtured, student driven." The members of the team do a majority of the work, the mentors are only there to provide guidance when it's needed. Each year the students are challenged to create a robot that can compete in a game. The robot that they create is made almost entirely from scratch; the students do everything from the designing to the programming. This competition provides high school-age kids with professional level experiences in a fun and hands-on way.
+<img class="float-left" alt="FRC Logo" style="width:275px;" src="{{ site.url }}/assets/img/FIRST-logos/FRC-logo.png"/> *FIRST* Robotics Competition is for kids from grades 9-12. While each team has students as its members, they also have coaches and mentors who help as needed. A motto created by our team that encapsulates this idea is "Mentor nurtured, student driven." The members of the team do a majority of the work, the mentors are only there to provide guidance when it's needed. Each year the students are challenged to create a robot that can compete in a game. The robot that they create is made almost entirely from scratch; the students do everything from the designing to the programming. This competition provides high school-age kids with professional level experiences in a fun and hands-on way. Read about some fast facts below! 
```

Comment:
Note: I suggested a change to the end of the paragraph. You might have to scroll to the end to see it.

Reply:
Should we just rephrase the paragraph and delete the part about "Click here for more"? I just don't think it's useful. We already described below in details what FRC is

Reply:
The paragraph could definitely do with some rephrasing, but don't get rid of the "Click here for more." It's always good to give a link for more information, especially since we're giving a very vague description in the paragraph. Linking to the official site is good since they'll always have the correct and most detailed information.